tests/toolchains/defs.bzl

```diff
@@ -24,36 +25,56 @@ powershell.exe -c "& ./{interpreter_path} {run_acceptance_test_py}"
 """
 
 def _acceptance_test_impl(ctx):
-    workspace = ctx.actions.declare_file("/".join([ctx.attr.python_version, "WORKSPACE"]))
-    ctx.actions.expand_template(
-        template = ctx.file._workspace_tmpl,
-        output = workspace,
-        substitutions = {"%python_version%": ctx.attr.python_version},
-    )
+    files = []
+
+    if BZLMOD_ENABLED:
+        module_bazel = ctx.actions.declare_file("/".join([ctx.attr.python_version, "MODULE.bazel"]))
```

Comment:
Ahhh, was the main problem that, bazel@head has --enable_bzlmod defaulted to true, so then the inner bazel finally picked that up, but then fell over because it didn't have a module file? That makes a lot of sense once pointed out.

Reply:
Yeah, it was just inheriting the `MODULE.bazel` from the `rules_python` and using the default toolchain from there.